src/custom/pages/Claim/ClaimsTable.tsx

```diff
@@ -109,6 +118,10 @@ export default function ClaimsTable({
   const hideTable =
     isAirdropOnly || !hasClaims || !activeClaimAccount || claimStatus !== ClaimStatus.DEFAULT || isInvestFlowActive
 
+  const start = useDeploymentTimestamp()
+  const investmentEnd = useInvestmentDeadline()
+  const airdropEnd = useAirdropDeadline()
```

Comment:
ubernit: probably these 3 related stuff could have been done in the same hook which return the 3 things. At the end of the day depends on the same thing (the deployment time).

not from this PR though

Reply:
https://github.com/gnosis/cowswap/pull/2179